cantor-misc/src/main/java/com/salesforce/cantor/misc/CantorProperties.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020, Salesforce.com, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+package com.salesforce.cantor.misc;
+
+import java.util.Map;
+
+public class CantorProperties {
```

Comment:
@AlexanderKouthoofd don't need this class.

Reply:
Removed.